ui/components/component-library/badge-wrapper/badge-wrapper.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
Should we convert these new files to typescript now?

Reply:
Will this work if it's using the `Box` component as a base?